traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -31,22 +34,39 @@ open class AvailabilityViewController: UIViewController {
     }
     var selectedOption: BookingOption?
     var optionsViewController: BookingOptionsViewController?
+    var numberOfMonths = 12
```

Comment:
This does not belong here! 1) Its bad practice to hard code values that are potentially configurable (like the number of months you want to display in a calendar) 2) This is calendar logic and it belongs in a different view controller.